tests/conftest.py

```diff
@@ -95,6 +95,20 @@ def pytest_configure(config):
         settings.STATIC_ROOT = os.path.join(os.path.dirname(rest_framework.__file__), 'static-root')
         settings.STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'
 
+    # Test that serializer fields can be created before django is set-up.
```

Comment:
I added this regression test in order to alert someone who looks at the weird lambdas and tries to remove them. I thought it better than to add a comment each time.

Reply:
This could be moved to `tests.importable`. Basically, it's a fake app that ensures parts of DRF can be imported before Django is fully setup. The tests just ensure that the various imports have occurred in the app `__init__`.